### ISSUE TITLE
Error on generated Adapter unit tests

### ISSUE THREAD

**Issue by pkenway** [dustinfarris/ember-django-adapter#79]
I'm getting an error running unit tests for the DRF adapter and subclassed adapters.

See the following gist for an example:
https://gist.github.com/pkenway/9e77ca425d91fe925482

This is the error I get:
```

not ok 1 PhantomJS 1.9 - ApplicationAdapter: it exists
    ---
        actual: >
            null
        message: >
            beforeEach failed on it exists: 'undefined' is not a function (evaluating '(function () {
                  return ENV['default'].APP.API_HOST;
                }).property()')
        Log: >
    ...
ok 2 PhantomJS 1.9 - JSHint - unit/adapters: unit/adapters/application-test.js should pass jshint
```

This is with ember-cli 0.2.2.

Am I doing something wrong or is this a bug?

**Comment by pkenway** [comment]
I figured out the issue.  I was including the adapter in an addon, which by default has the prototype extensions disabled.  I uninstalled `ember-disable-prototype-extensions` and it's fixed.

**Comment by dustinfarris** [comment]
Good to know.  This sounds like something we should document.

**Comment by benkonrath** [comment]
Ember-cli 0.2.4 includes ember-disable-prototype-extensions by default so we should probably figure out why the tests don't work when the prototype extensions are turned off. 

Here's some information about the module:
> This is a great idea for addon authors to ensure that their addon does not accidentally
> depend upon prototype extensions (which may be disabled in a consuming application).

https://github.com/rwjblue/ember-disable-prototype-extensions  

The PR into ember-cli also has some useful information:
https://github.com/ember-cli/ember-cli/pull/3645

**Comment by dustinfarris** [comment]
ok.  I'm guessing 0.2.5 is right around the corner with today's Ember version bumps.  When that happens one of us should start a PR to bump our required version—and fix this issue while we're at it.

**Comment by dustinfarris** [comment]
deleted last comment, got my cli versions mixed up.

Yes, we need to update to 0.2.4 and fix this issue.

**Comment by benkonrath** [comment]
Do you want me to investigate / fix this issue before you merge #89?

**Comment by dustinfarris** [comment]
Yes, thanks.
